### PR TITLE
ITNFA-30: Extended patientSearchWidget

### PR DESCRIPTION
**Pull request by dserkowski**

### CODE REVIEW

File: api/src/main/resources/messages.properties [hunk 1]
```diff
@@ -447,6 +447,7 @@ coreapps.search.identifier=Identifier
 coreapps.search.name=Name
 coreapps.search.info=Showing _START_ to _END_ of _TOTAL_ entries
 coreapps.search.label.recent=Recent
+coreapps.search.label.onlyInMpi=Only in MPI
```
Comment (pawelcieszko) [inline api/src/main/resources/messages.properties]: I'm not sure if 'only in MPI is good label for end-user

Comment (pawelcieszko) [reply api/src/main/resources/messages.properties]: Maybe something like: 'remote', 'external', 'not imported'

Comment (dserkowski) [reply api/src/main/resources/messages.properties]: I think 'not imported' will be the best one. However, I will send an email to Nathaelf about this.

Comment (pawelcieszko) [reply api/src/main/resources/messages.properties]: Right, I'm looking forward to his response.

**Pull request merged**